odoo/tests/__init__.py

```diff
@@ -8,4 +8,5 @@
 from . import common
 from .common import *
 from .form import Form, O2MProxy, M2MProxy
+from . import test_limited_field_access_token
```

Comment:
business tests are typically in addons instead (see `test_05_web_image_access_token`)

We might just want an extra assert there for a token signed for a different scope not allowing access to the route